Improves space RP.

Pickle-Coding opened the pull request:
## About The Pull Request
Improves space RP by replacing instances of tiles referring to length and volume with meters and liters.

## Why It's Good For The Game
Improves space RP. Especially beneficial when the context is in space and people don't have tiles as a reference.

## Changelog

:cl:
spellcheck: Replaces instances of tiles referring to length and volume with meters and liters.
/:cl:

Pickle-Coding committed:
Improves space RP.

Improves space RP by replacing instances of tiles referring to length and volume with meters and liters.

Pickle-Coding committed:
Revert replacing instances in an admin tool.

Admins don't need space RP.

LemonInTheDark reviewed (changes requested):
I worry this will harm parsability. it is more correct, but it also is another "jump" you have to make

mrmanlikesbt commented on code/datums/shuttles/escape_pod.dm:
```diff
@@ -5,7 +5,7 @@
 	suffix = "default"
 	name = "escape pod (Default)"
-	description = "Base escape pod with 2 tiles of interior space."
+	description = "Base escape pod with 5 kilo liters of interior space."
```
why you doin liters instead of square meters?

SmArtKar replied:
```suggestion
	description = "Base escape pod with 2 square meters of interior space."
```

SmArtKar commented on code/datums/shuttles/escape_pod.dm:
```diff
@@ -15,7 +15,7 @@
 	suffix = "luxury"
 	name = "escape pod (Luxury)"
-	description = "Upgraded escape pod with 3 tiles of interior space."
+	description = "Upgraded escape pod with 7.5 kilo liters of interior space."
```
```suggestion
	description = "Upgraded escape pod with 3 square meters of interior space."
```

SmArtKar reviewed (changes requested):
I actually like this a lot in term of vibes but lemon's concerns are also valid

Pickle-Coding committed:
Liters to cubic meters.

People are more familiar with that unit.

necromanceranne reviewed (changes requested):
This is practically incomprehensible to me, space and distance is abstracted as fuck. How the fuck is a player supposed to interpret metres in actual game distance? How is that a useful tooltip?

Pickle-Coding committed:
Revert for pods.

Pods no longer use cubic meters as the only reference for the height of the ceiling in this game is deriving it from the volume the gas analyzer reports when scanning the air.

LemonInTheDark commented:
just do it as a comment as a rule, most contributors don't know who all the maintainers are because NOT EVERYONE HAS MEMBER SET TO PUBLIC (and issue managers can do the same thing) and you will just confuse people.

JohnFulpWillard commented:
can't we make the text underlined and hovering over shows the OOC numbers

Pickle-Coding reviewed (commented):
> can't we make the text underlined and hovering over shows the OOC numbers 

Good idea. I'll need to find out how to do that first.

Exester509 commented:
This not only makes measures much more abstract, it also conflicts with the actual defined litre we have, used in various atmos machines.
There's a reason why chemistry deals in ambiguous "units" instead of actual volumes as well

necromanceranne commented:
> ```dm
> "Transmission 4" = "Causes the host to periodically emit a yawn that tries to infect bystanders within 6 meters of the host."
> ```
> 
> ```dm
> "Transmission 7" = "Coughing will now infect bystanders up to 2 tiles away.",
> ```
> 
> Not even consistent about it in the same ui man

Funking LOL?

Pickle-Coding committed:
Meters tooltip.

Adds a tooltip for meters to display it as tiles. Uses it for instances of meters in spans and to_chats.

Pickle-Coding reviewed (commented):
> can't we make the text underlined and hovering over shows the OOC numbers 

I replaced instances of it found in the span procs and to_chat(). Sadly it counted as a non-constant expression when I tried to use it for desc.

Pickle-Coding reviewed (commented):
> Doesn't this mean that it makes all spacemen around 1 meter tall by association with the tile size? that's just about over 3 feet for people who use that, that's short as fuck! 

Tiles are 1 meter long but that doesn't mean the ceiling is 1 meter high. They're actually 2.5 meters high.

Pickle-Coding reviewed (commented):
> This does bring up an interesting question: What is a chemical unit? An average drinking glass holds 50 units of the stuff, so I'd assume based on a real-world standard being 250 ml, our glasses are the same size. This means we could hypothetically replace 1 unit with 50 milliliters. However, this would probably become very confusing for any decimalized points, so I think it's better to keep them as abstract units. 

Units are used both for volume (reagent containers have a fixed unit limit) and quantity (reactions are based around units). I think that will cause additional confusion if we chose either real unit.

LemonInTheDark commented:
a u of liquid is a mole (counting unit) IE 6.02214076×10^23 atoms of equivalent gas. Hope that helps!

Pickle-Coding committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into meters
